po/de.po

```diff
@@ -748,201 +743,250 @@ msgid "With Java"
 msgstr "Mit Java"
 
 #: src/SUMMARY.md:229
+msgid "Day 4: Afternoon (Async)"
+msgstr "Tag 4: Nachmittags (Async)"
+
+#: src/SUMMARY.md:233
+msgid "Async Basics"
+msgstr "Async Grundlagen"
+
+#: src/SUMMARY.md:234
+msgid "async/await"
+msgstr ""
+
+#: src/SUMMARY.md:235
+msgid "Futures"
+msgstr ""
+
+#: src/SUMMARY.md:236
+msgid "Runtimes"
+msgstr "Laufzeiten"
+
+#: src/SUMMARY.md:237
+msgid "Tokio"
+msgstr ""
+
+#: src/SUMMARY.md:238
+msgid "Tasks"
+msgstr "Aufgaben"
+
+#: src/SUMMARY.md:239
+msgid "Async Channels"
+msgstr "Async Kanäle"
+
+#: src/SUMMARY.md:241
+msgid "Join"
+msgstr ""
+
+#: src/SUMMARY.md:242
+msgid "Select"
+msgstr ""
+
+#: src/SUMMARY.md:243
+msgid "Pitfalls"
+msgstr "Tücken"
+
+#: src/SUMMARY.md:244
+msgid "Blocking the Executor"
+msgstr "Blockieren des Ausführers"
+
+#: src/SUMMARY.md:245
+msgid "Pin"
+msgstr ""
+
+#: src/SUMMARY.md:246
+msgid "Async Traits"
+msgstr "Async Merkmale"
+
+#: src/SUMMARY.md:249
 msgid "Final Words"
 msgstr "Letzte Worte"
 
-#: src/SUMMARY.md:231
+#: src/SUMMARY.md:251
 msgid "Thanks!"
 msgstr "Danke!"
 
-#: src/SUMMARY.md:232
+#: src/SUMMARY.md:252
 msgid "Other Resources"
 msgstr "Andere Ressourcen"
 
-#: src/SUMMARY.md:233
+#: src/SUMMARY.md:253
 msgid "Credits"
 msgstr "Würdigungen"
 
-#: src/SUMMARY.md:237
+#: src/SUMMARY.md:257
 msgid "Bare Metal Rust: Morning"
-msgstr ""
+msgstr "Hardwarenahes Rust: Morgens"
 
-#: src/SUMMARY.md:240
+#: src/SUMMARY.md:260
 msgid "no_std"
 msgstr ""
 
-#: src/SUMMARY.md:241
-#, fuzzy
+#: src/SUMMARY.md:261
 msgid "A Minimal Example"
-msgstr "# Kleines Beispiel"
+msgstr "Kleines Beispiel"
 
-#: src/SUMMARY.md:242
+#: src/SUMMARY.md:262
 msgid "alloc"
 msgstr ""
 
-#: src/SUMMARY.md:243
+#: src/SUMMARY.md:263
 msgid "Microcontrollers"
-msgstr ""
+msgstr "Mikrokontroller"
 
-#: src/SUMMARY.md:244
+#: src/SUMMARY.md:264
 msgid "Raw MMIO"
-msgstr ""
+msgstr "MMIO"
 
-#: src/SUMMARY.md:245
+#: src/SUMMARY.md:265
 msgid "PACs"
 msgstr ""
 
-#: src/SUMMARY.md:246
+#: src/SUMMARY.md:266
 msgid "HAL Crates"
-msgstr ""
+msgstr "HAL Kisten"
```

Comment:
You should decide if you want to translate "crate" to "Kiste" or not? I would personally not translate it, but it's mostly up to you (and other German speakers in the community).

Reply:
When looking at the german translation of the Rust Book, they used "Kisten (crates)" as a translation. However, I think it makes sense to just stick with "crates" too, since this is used commonly. So from here on, I would just use "crates" and when it makes sense "crates (Kisten)".